central/test-framework/org.jboss.tools.central.reddeer/src/org/jboss/tools/central/reddeer/wizards/JBossCentralProjectWizardLauncherApplication.java

```diff
@@ -0,0 +1,231 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.central.reddeer.wizards;
+
+import org.eclipse.reddeer.common.logging.Logger;
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitUntil;
+import org.eclipse.reddeer.common.wait.WaitWhile;
+import org.eclipse.reddeer.jface.wizard.WizardDialog;
+import org.eclipse.reddeer.swt.api.Button;
+import org.eclipse.reddeer.swt.condition.ShellIsActive;
+import org.eclipse.reddeer.swt.impl.browser.InternalBrowser;
+import org.eclipse.reddeer.swt.impl.button.CheckBox;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.combo.DefaultCombo;
+import org.eclipse.reddeer.swt.impl.shell.DefaultShell;
+import org.eclipse.reddeer.swt.impl.text.LabeledText;
+import org.eclipse.reddeer.workbench.core.condition.JobIsRunning;
+import org.jboss.tools.central.reddeer.api.JavaScriptHelper;
+import org.jboss.tools.central.reddeer.projects.ArchetypeProject;
+import org.jboss.tools.central.reddeer.wait.CentralIsLoaded;
+
+
+/**
+ * 
+ * @author zcervink
+ *
+ */
+
+
+public class JBossCentralProjectWizardLauncherApplication extends WizardDialog {
```

Comment:
Class name should be same as it is in original Launcher class: https://github.com/jbosstools/jbosstools-central/blob/master/launcher/plugins/org.jboss.tools.common.launcher.ui/src/main/java/org/jboss/tools/common/launcher/ui/wizard/NewLauncherProjectWizard.java

Reply:
I would also suggest to separete this file in two files: xxxWizard.java and xxxWizardPage.java to have same hierarchy as it is in launcher plugin

Reply:
I have split the class into 2 classes.

Reply:
Opening Launcher Application Wizard is now working using the "NEW" dialog. Tested on localhost, now testing on Jenkins.